src/main/java/com/segment/analytics/android/integrations/google/analytics/GoogleAnalyticsIntegration.java

```diff
@@ -135,6 +138,20 @@
 
     ScreenViewHitBuilder hitBuilder = new ScreenViewHitBuilder();
     attachCustomDimensionsAndMetrics(hitBuilder, properties);
+
+    AnalyticsContext context = screen.context();
```

Comment:
If you pull this into a method (like `attachCustomDimensionsAndMetrics` we can do this for track calls too) with just one extra line.